### PR TITLE
Bebosudo pep8 pep257

### PR DESCRIPTION
**Pull request by bebosudo**
I applied pylint and fixed some issues on all the interface files, like lines longer than 80 chars, or single-line docstrings on multiple lines.

There are still some issues according to pylint:
```
$ pylint --rcfile=pylint.rc bitex/ 
Using config file bitex/pylint.rc
************* Module bitex.interface.bittrex
W: 99, 4: Keyword argument before variable positional arguments list in the definition of wallet function (keyword-arg-before-vararg)
************* Module bitex.interface.binance
W: 61, 0: Unused argument 'args' (unused-argument)
************* Module bitex.api.REST.itbit
R: 61, 4: Too many local variables (16/15) (too-many-locals)
************* Module bitex.api.REST.quoine
E: 12, 0: Unable to import 'jwt' (import-error)
************* Module bitex.api.REST.binance
W: 51, 4: Parameters differ from overridden 'sign_request_kwargs' method (arguments-differ)

------------------------------------------------------------------
Your code has been rated at 9.96/10 (previous run: 9.95/10, +0.01)
```

### CODE REVIEW

File: bitex/exceptions.py [hunk 1]
```diff
@@ -17,23 +17,27 @@ class IncompleteCredentialsWarning(UserWarning):
 
 
 class IncompleteCredentialConfigurationWarning(UserWarning):
-    """Raised when a required parameter for authentication isn't found in a config file."""
+    """Raised when a required parameter for authentication isn't found in a
```
Comment (nlsdfnbch) [inline bitex/exceptions.py]: this is invalid - Docstrings without parameters should be a single line. (PEP257)

File: bitex/exceptions.py [hunk 2]
```diff
@@ -17,23 +17,27 @@ class IncompleteCredentialsWarning(UserWarning):
...
 
     pass
 
 
 class IncompleteCredentialsError(Exception):
     """Raised when attempting to call private_query() of an API.
 
-    For example when missing essential values to create a message signature (key, secret
-    or additional other values, dependent on the exchange).
+    For example when missing essential values to create a message signature
```
Comment (nlsdfnbch) [inline bitex/exceptions.py]: this is invalid - Docstrings without parameters should be a single line. (PEP257)

File: bitex/interface/binance.py [hunk 3]
```diff
@@ -23,47 +19,40 @@ class Binance(RESTInterface):
...
-                                             **req_kwargs)
+        return super(Binance, self).request(verb, endpoint,
+                                            authenticate=authenticate,
+                                            **req_kwargs)
 
     def _get_supported_pairs(self):
-        """
-        Generate a list of supported pairs.
-        """
+        """Return a list of supported pairs."""
```
Comment (nlsdfnbch) [inline bitex/interface/binance.py]: Good catch! But the blank lines after the comment should be removed as well.

**Comment by bebosudo** [comment]
> it should be fine if you only used pylint with the rcfile
oops. the problem is that I used an external plugin of my text editor.

File: bitex/interface/ccex.py [hunk 4]
```diff
@@ -26,13 +23,15 @@ def request(self, endpoint, authenticate=False, **req_kwargs):
...
+            return super(CCEX, self).request('GET', endpoint,
+                                             authenticate=True, **req_kwargs)
+
         endpoint = endpoint if endpoint else 'api_pub.html'
         return super(CCEX, self).request('GET', endpoint, **req_kwargs)
 
     def _get_supported_pairs(self):
         """Return a list of supported pairs."""
-        return requests.get('https://c-cex.com/t/pairs.json').json()['pairs']
+        return self.request('/pairs.json').json()['pairs']
```
Comment (nlsdfnbch) [inline bitex/interface/ccex.py]: 👍

**Comment by nlsdfnbch** [comment]
>Feel free to drop this PR.
I'll wait for your changes - if you'd rather submit a new PR, then I'll close this one. Thanks for your efforts! :)

**Comment by bebosudo** [comment]
I'll submit a new PR with the same changes except the >80 long line edits.

**Pull request closed without merging**